src/app/fdctl/run/tiles/fd_pack.c

```diff
@@ -168,6 +171,10 @@ during_housekeeping( void * _ctx ) {
     ctx->poh_reset_slot    = new_reset_slot;
     ctx->poh_slots_updated = 1; /* Handle all the state transitions in before_credit below */
   }
+
+  /* Update metrics */
```

Comment:
Might be better to use `metrics_write` callback for this, it has a COMPILER_MFENCE